aws/ondemand/main.sh

```diff
@@ -24,6 +24,7 @@ ok
 log "Initial system configuration..."
 chmod a-x /etc/update-motd.d/*
 mkdir -p /home/ubuntu/.cache
+chown ubuntu:ubuntu /home/ubuntu/.cache
```

Comment:
is this related?

Reply:
oops not related, but I noticed random commands failing because they can't write there so I fixed that as well